cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -1535,3 +1540,51 @@ export function redrawShapeAsync(): ThunkAction {
         }
     };
 }
+
+export function trackerSettings(name: string, value: string | number): AnyAction {
+    return {
+        type: AnnotationActionTypes.TRACKER_SETTINGS,
+        payload: {
+            name,
+            value,
+        },
+    };
+}
+
+export function resetTrackerSettings(): AnyAction {
+    return {
+        type: AnnotationActionTypes.RESET_TRACKER_SETTINGS,
+    };
+}
+
+export function doTracking(body: any, taskId: string): ThunkAction {
+    return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
+        dispatch({
+            type: AnnotationActionTypes.DO_TRACKING,
+        });
+        const baseUrl = cvat.config.backendAPI.slice(0, -7);
+        try {
+            cvat.server.request(`${baseUrl}/api/v1/tasks/${taskId}/tracking/track`, {
+                method: 'POST',
+                data: JSON.stringify(body),
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+            }).then((response: any) => {
+                console.log(response)
```

Comment:
Let's remove console.log from the patch. It looks like made for debug

Reply:
Done.